packages/web-components/@sfx/sayt/package.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "@sfx/sayt",
+  "version": "0.0.0",
+  "description": "",
+  "main": "./src/index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "../../../../scripts/build.sh",
+    "dev": "nodemon --watch src --exec npm run build"
+  },
```

Same feedback as the logic layer as per https://docs.npmjs.com/files/package.json#repository
```suggestion
  },
  "repository": {
  "type" : "git",
  "url" : "https://github.com/groupby/sfx-view.git",
  "directory": "packages/web-components/sayt"
},
```

---

Oh cool, I didn't know about this. I will replicate this in both repos.

---

Updated.